Vision/Vision.bs

```diff
@@ -86,7 +86,7 @@ Boilerplate: omit conformance
 	as it expands reach, knowledge, education, and services even more broadly.
 	We believe the World Wide Web should be 
 	inclusive and respectful of its users: 
-	a Web that supports truth over falsehood, 
+	a Web that supports facts over falsehood, 
```

Comment:
minor, they should be both plural or both singular, and it's punchier as "fact over falsehood". Perhaps that it's a direction of optimization would be clearer if it said "promotes" or "prefers" or "values"

Reply:
made both plural.